lib/evss/service.rb

```diff
@@ -36,13 +36,25 @@ def headers_for_user(user)
       EVSS::AuthHeaders.new(user).to_h
     end
 
+    def save_error_details(error)
```

Comment:
Would there be value in tagging the calling service here?
e.g. 
```suggestion
    def save_error_details(error, service_name)
      Raven.tags_context(service: service_name)
```

And then call this method from inside the subclasses as appropriate.  e.g.
`save_error_details(error, 'ppiu')`